sphinx/source/docs/reference/server/views.rst

```diff
@@ -1,10 +1,10 @@
 bokeh.server.views
 ------------------
 
-auth_mixin
+auth_request_handler
 ~~~~~~~~~~
```

Comment:
Sphinx is picky, this underline needs to be exactly the same length as `auth_request_handler`

Reply:
Got it, thanks for the help.